src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+
+import { IoTHubManagerService } from 'services';
+import {
+  copyToClipboard,
+  LinkedComponent,
+  svgs
+} from 'utilities';
+import { Svg } from 'components/shared/svg/svg';
+import {
+  Btn,
+  BtnToolbar,
+  ErrorMsg,
+  Flyout,
+  FlyoutHeader,
+  FlyoutTitle,
+  FlyoutCloseBtn,
+  FlyoutContent,
+  FormSection,
+  Indicator,
+  SectionDesc,
+  SectionHeader,
+  SummaryCount,
+  SummarySection
+} from 'components/shared';
+
+import './deviceNew.css';
+
+const DeviceDetail = ({ label, value }) => (
+  <FormSection className="device-detail">
+    <SectionHeader>{label}</SectionHeader>
+    <div className="device-detail-contents">
+      <div className="device-detail-value">{value}</div>
+      <Svg className="copy-icon" path={svgs.copy} onClick={() => copyToClipboard(value)} />
+    </div>
+  </FormSection>
+);
+
+const DeviceConnectionString = ({ label, deviceId, hostName, sharedAccessKey }) => (
+  <DeviceDetail label={label} value={`HostName=${hostName};DeviceId=${deviceId};SharedAccessKey=${sharedAccessKey}`} />
+);
+
+
+const ProvisionedDevice = ({ device, t }) => {
+  const { id, ioTHubHostName: hostName, authentication: { primaryKey }, authentication: { secondaryKey } } = device;
+
+  return (
+    <div>
+      <pre>{JSON.stringify(device, null, 2)}</pre>
```

Comment:
Why is this here? It is a left over?

Reply:
Because this page is not done yet... and so yes, its temporary. The next version of this branch won't have it.